ee-9/common/src/main/resources/license/preview-feature-pack-common-licenses.xml

```diff
@@ -2,545 +2,19 @@
 <licenseSummary>
   <dependencies>
     <dependency>
-      <groupId>${project.groupId}</groupId>
-      <artifactId>wildfly-ee-9-deployment-transformer</artifactId>
-      <licenses>
-        <license>
-          <name>GNU Lesser General Public License v2.1 or later</name>
-          <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html</url>
-          <distribution>repo</distribution>
-        </license>
-      </licenses>
-    </dependency>
```

Comment:
This entry should be kept.

Reply:
There are a number of dependency entries in https://github.com/wildfly/wildfly/blob/main/ee-9/common/pom.xml that now don't have matching entries in this file. They all should.